src/components/HomeStatistics.jsx

```diff
@@ -1,42 +1,105 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchPhaseTwoWorkflows } from '../ducks/workflow';
+import { config } from '../config';
 import Connect from './Connect';
 
-export default function HomeStatistics() {
-  return (
-    <div className="statistics">
-      <div className="statistics__stat-content">
-        <div className="statistics__pie-chart">
-          <svg viewBox="0 0 200 200">
-            <circle cx="100" cy="100" r="100" fill="#EBDFD6" stroke="#472B36" strokeWidth="5px" />
-            <path d="M100 100 L 100,0 A100,100 0 0,1 200,100 z" fill="#472B36" />
-          </svg>
-          <span>Percent Complete</span>
-          <hr className="plum-line" />
-        </div>
-        <div className="statistics__statistics">
-          <h2>Statistics from phase two of Scribes of the Cairo Geniza</h2>
-          <div className="statistics__stat">
-            <div>
-              <span>831</span>
-              <span>Volunteers</span>
-              <hr className="plum-line" />
-            </div>
-            <div>
-              <span>12,473</span>
-              <span>Classifications</span>
-              <hr className="plum-line" />
-            </div>
-            <div>
-              <span>3,147</span>
-              <span>Completed Subjects</span>
-              <hr className="plum-line" />
+class HomeStatistics extends React.Component {
+  componentDidMount() {
+    this.props.dispatch(fetchPhaseTwoWorkflows());
+  }
+
+  getCoordinatesForPercent(percent) {
+    const x = Math.cos(2 * Math.PI * percent);
+    const y = Math.sin(2 * Math.PI * percent);
+    return { x, y };
+  }
+
+  render() {
+    const { project, allWorkflows } = this.props;
+    if (!project || !allWorkflows) return null;
+    let classificationsCount = 0;
+    let percentComplete = 0;
+    let retiredSubjects = 0;
+    let subjectsCount = 0;
+
+    Object.keys(allWorkflows).map((id) => {
```

Comment:
If `allWorkflows` is an array to begin with you avoid converting back to an array by calling `Object.keys()`. `forEach` would be more appropriate to use here since you don't need to return an array.

Reply:
`allWorkflows` is actually an object. It is kept an object so it is easier to retrieve a different workflow (English, Arabic, Hebrew) throughout the app by using literal notation with the workflow id.